site/realm/functions/createDefaultAdminUser.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * This function creates the Default Admin User that is used run the tests
+ * 
+ * @param {string} email The email address of the user
+ * @param {string} password The password of the user
+ * 
+*/
+exports = async (input) => {
+
+    //check if the Admin user is already created
+    const usersCollection = context.services.get('mongodb-atlas').db('customData').collection("users");
```

Comment:
to be consistent on string style, stay with single quotes (')

Reply:
Done